src/app/core/feedback/feedback.controller.spec.js

```diff
@@ -152,20 +145,73 @@ describe('Feedback Controller', () => {
 					.expect('Content-Type', /json/)
 					.expect(testConfig.isAdmin ? 200 : 403)
 					.expect((res) => {
-
 						if (testConfig.isAdmin) {
 							const expected = savedFeedback.toJSON();
 							expected._id = expected._id.toString();
 
 							should(res.body).eql({
-								elements: [ expected ],
+								elements: [expected],
 								pageNumber: 0,
 								pageSize: 20,
 								totalPages: 1,
 								totalSize: 1
 							});
+						} else {
+							should(res.body).eql({
+								message: 'This is a fake error message'
+							});
 						}
-						else {
+					})
+					.end(done);
+			});
+		});
+	});
+
+	describe('PATCH /admin/feedback/:feedbackId/status', () => {
+		let savedFeedback;
+
+		before(() => {
+			return Feedback.deleteMany({})
+				.exec()
+				.then(() => {
+					return new Feedback({
+						body: 'testing',
+						url: 'http://localhost:3000/home',
+						type: 'Question'
+					})
+						.save()
+						.then((result) => (savedFeedback = result));
+				});
+		});
+
+		[
+			{
+				name: 'should not update feedback status as non-admin',
+				isAdmin: false
+			},
+			{ name: 'should update feedback status as admin', isAdmin: true }
+		].forEach((testConfig) => {
+			it(testConfig.name, (done) => {
+				const updatedStatus = 'Closed';
+				setAdmin(testConfig.isAdmin);
+				request(app)
+					.patch(`/admin/feedback/${savedFeedback._id}/status`)
+					.send({ status: updatedStatus })
+					.expect('Content-Type', /json/)
+					.expect(testConfig.isAdmin ? 200 : 403)
+					.expect((res) => {
+						if (testConfig.isAdmin) {
+							const expected = savedFeedback.toJSON();
+							expected._id = expected._id.toString();
+							expected.status = updatedStatus;
+							// Verify that the 'updated' field was updated
```

Comment:
Is this an acceptable way of testing that the `updated` field was updated? Is there a better/more straightforward way of doing this?

Reply:
LGTM! You could check that the updated date is greaterThan the previous date, but lessThanOrEqual to now ( a new Date ), but this seems reasonable.